dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -429,11 +428,11 @@ public void launch(final List<String> args) {
 
                         switch (descriptor) {
                         case CWL_STRING:
-                            if (!(yamlRun != null ^ jsonRun != null ^ tsvRun != null)) {
+                            if (!(yamlRun != null ^ jsonRun != null)) {
```

Comment:
What is happening to tsvRun? Is support for it being dropped?

Reply:
This branch originally started as a removal of cwltool for cromwell, which does not support tsv. I can add tsv back in.

Reply:
I think we can drop tsvRun actually, I suspect it doesn't get much use